pyproject.toml

```diff
@@ -39,6 +39,12 @@ pre-commit = "^2.15.0"
 pytest-cov = "^3.0.0"
 setuptools-rust = "^1.1.2"
 mkdocs-material = "8.5.10"
+mkdocstrings = "^0.19.1"
+
+[tool.poetry.group.dev.dependencies]
```

Comment:
I see this has been made with a newer version of Poetry which [handles groups](https://python-poetry.org/docs/managing-dependencies/#dependency-groups). Maybe it's working fine, but I would still move dev-depencencies to the newer group section to avoid any problems.